apps/backend/app/services/feature_store_service.py

```
@@ -0,0 +1,715 @@
+        """Check if user has access to feature."""
+        return (
+            feature.user_id == user_id or
+            feature.is_public or
+            user_id in feature.shared_with
+        )
+
+    async def search_features(
+        self,
+        user_id: str,
+        query: Optional[str] = None,
+        filters: Dict[str, Any] = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)